contracts/liquidity_pool.cairo

```diff
@@ -413,6 +413,11 @@ func deposit_liquidity{syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_che
         set_lpool_balance(lptoken_address, new_pb);
     }
 
+    with_attr error_message("lpool balance exceeds set maximum"){
```

Comment:
please add a comment why this is needed (what it is used for)... the inside... the max_balance cap...